### PR TITLE
Allow usage of hack/build/build.sh when not in a git repo

### PR DESCRIPTION
**Pull request by abardavi**
Signed-off-by: Ariel (Ori) Bar David <abardavi@redhat.com>

```release-note
Allow usage of hack/build/build.sh when not in a git repo
```

### CODE REVIEW

File: hack/build/build.sh [hunk 1]
```diff
@@ -9,15 +9,19 @@ fi
 
 TAG="$1"  #TODO: validate tag is vX.Y.Z
 COMPONENT="kubevirt-template-validator"
-BRANCH=$( git rev-parse --abbrev-ref HEAD )
-REVISION=$( git rev-parse --short HEAD )
+LDFLAGS="\
+-X 'github.com/fromanirh/kubevirt-template-validator/internal/pkg/version.COMPONENT=$COMPONENT' \
```
Comment (ksimon1) [inline hack/build/build.sh:13]: please use `github.com/kubevirt` not `github.com/fromanirh`

**Review by ksimon1 — CHANGES_REQUESTED** [review]

**Comment by eedri** [comment]
@abardavi can you send an update?

File: hack/build/build.sh (hunk 1 above)
Comment (abardavi) [reply hack/build/build.sh:13]: If I do that the variables do not get set.
IIUC in order to do have that change, there must be more changes that are out of the scope of this PR.

**Pull request merged**